src/client/opamArg.ml

```diff
@@ -1983,7 +1983,7 @@ let check_and_run_external_commands () =
       && List.for_all (fun (_,info) -> Term.name info <> name) commands
     then
     (* No such command, check if there is a matching plugin *)
-    let command = opam ^ "-" ^ name in
+    let command = (Filename.basename opam) ^ "-" ^ name in
```

Comment:
> That's effectively what it is, unless the opam command itself is named something else.

Precisely it seems to me you don't want the plugins to fail if `opam` happens to be named differently.